citeproc/model.py

```diff
@@ -104,7 +103,7 @@ def get_term(self, name, form=None):
             for locale in self.get_root().locales:
                 try:
                     return locale.get_term(name, form)
-                except IndexError: # TODO: create custom exception
+                except IndexError:  # TODO: create custom exception
```

Comment:
The TODO says this should be a separate exception; does that mean locale.get_term should be raising a custom IndexError subclass?